webpack.config.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
                     {
                       pattern: /%%PATH_TO_CONTROLLER%%/g,
                       replacement: function (match, p1, offset, string) {
-                        return controllerPaths[conf.pwt.adserver || "DFP"];
+                        console.log("***** Returning controller path as -> "+controllerPaths[adserver]+" and adserver = " + adserver);
+                        return controllerPaths[adserver];
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 8 tabs but found 24 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14712)